packages/plugin-ext/src/common/plugin-protocol.ts

```diff
@@ -76,6 +77,13 @@ export interface PluginPackageView {
     name: string;
 }
 
+export interface PluginPackageCommand {
+    command: string;
+    title: string;
+    category?: string;
+    icon?: string | { light: string; dark: string; };
```

Comment:
hello again: why here do we have icon light and icon dark ?

Reply:
Should we ignore and don't parse from package.json? https://github.com/Microsoft/vscode/blob/366bfe96bf2fe4cff37de895c3fa2ccb776eb621/extensions/markdown-language-features/package.json#L29-L36

What is your suggestion?

Reply:
hello, my question is why it's not IconUrl  that you introduced there
```
export type IconUrl = string | { light: string; dark: string; };
```

Reply:
`PluginPackageCommand` is a format of raw command as it is defined by VS Code, with relative paths for icons. In order to download them, paths are converted into URLs served by `hostedPlugin` entry point. A new type is introduced to make it clear that a string is a url, not a raw path.

Reply:
shouldn't be then URI instead of string for iconUrl ?

Reply:
I thought about it, but then we need to transfer such urls from backend to frontend, and decided to stick with strings as for other remote APIs.